src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static final Task[] TASKS = new Task[101];
+    static int tasksI = 1;
+
     public static void main(String[] args) {
+        // print intro message
+        start();
+        // run bot (decode task)
+        run();
+    }
+
+    public static void start() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void list() {
+        System.out.println("Here are the tasks in your list: ");
+        for (int i = 1; i < tasksI; i++) {
+            Task currTask = TASKS[i];
+            System.out.println(i + ". " + currTask.toString());
+        }
+    }
+
+    public static void markUnmark(String word, Task task) {
+        if (word.equals("mark")) {
+            task.mark();
+            System.out.println("Nice! I've marked this task as done: ");
+            System.out.println(task.toString());
+        } else {
+            task.unmark();
+            System.out.println("OK, I've marked this task as not done yet: ");
+            System.out.println(task.toString());
+        }
+    }
```

Comment:
Perhaps you could remove the flag variables and create two new methods instead.
You could refer to this link for more: https://refactoring.com/catalog/removeFlagArgument.html